demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -169,16 +169,16 @@ def get_model(ie, args):
         raise ValueError("{} model doesn't support input transforms.".format(args.architecture_type))
 
     if args.architecture_type == 'ssd':
-        return models.SSD(*common_args, labels=args.labels, keep_aspect_ratio_resize=args.keep_aspect_ratio,
+        return models.SSD(*common_args, labels=args.labels, resize_type = 'keep_aspect_ratio' if args.keep_aspect_ratio else 'default',
                           threshold=args.prob_threshold)
     elif args.architecture_type == 'ctpn':
         return models.CTPN(ie, args.model, input_size=args.input_size, threshold=args.prob_threshold)
     elif args.architecture_type == 'yolo':
-        return models.YOLO(ie, args.model, labels=args.labels,
-                           threshold=args.prob_threshold, keep_aspect_ratio=args.keep_aspect_ratio)
+        return models.YOLO(ie, args.model, labels=args.labels, resize_type = 'letterbox' if args.keep_aspect_ratio else 'default',
+                           threshold=args.prob_threshold)
     elif args.architecture_type == 'yolov4':
         return models.YoloV4(ie, args.model, labels=args.labels,
-                             threshold=args.prob_threshold, keep_aspect_ratio=args.keep_aspect_ratio,
+                             threshold=args.prob_threshold, resize_type = 'letterbox' if args.keep_aspect_ratio else 'default',
```

Comment:
I already asked about this in internal conversation, but do not see fix for default resize type

Reply:
the question was that resize_type for aspect ratio depends from model architecture, but we select it outside. Is there is way to select default aspect ratio realisation inside model if user explicitly do not say which resize type use